pages/documentation/simulation/powerflow/openlf.md

```diff
@@ -2,4 +2,64 @@
 layout: default
 ---
 
-# Open LoadFlow
+# OpenLoadFlow
+
+PowSyBl OpenLoadFlow is an open-source power flow implementation in Java provided by PowSyBl. The source code is hosted on [GitHub](https://github.com/powsybl/powsybl-open-loadflow). 
+
+* TOC
+{:toc}
+
+## Configuration
+To use PowSyBl OpenLoadFlow for all power flow computations, you have to configure the `load-flow` module in your configuration file:
+```yaml
+load-flow:
+  default-impl-name: "OpenLoadFlow"
+```
+
+### Specific parameters
+
```

Comment:
I have to change here too parameters that are at ````LoadflowParameters```` level now. Can I commit in your PR ?